<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>ResizeWidget</class>
 <widget class="QWidget" name="ResizeWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>70</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" >
   <property name="margin" >
    <number>3</number>
   </property>
   <property name="spacing" >
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>4</number>
     </property>
     <item rowspan="2" row="1" column="3" >
      <widget class="QToolButton" name="btnLink" >
       <property name="sizePolicy" >
        <sizepolicy>
         <hsizetype>1</hsizetype>
         <vsizetype>0</vsizetype>
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip" >
        <string>Width and Height Linked</string>
       </property>
       <property name="text" >
        <string/>
       </property>
       <property name="icon" >
        <iconset resource="resize.qrc" >:/linked.png</iconset>
       </property>
       <property name="iconSize" >
        <size>
         <width>24</width>
         <height>9</height>
        </size>
       </property>
       <property name="autoRaise" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="label_4" >
       <property name="text" >
        <string>In Pi&amp;xels</string>
       </property>
       <property name="buddy" >
        <cstring>spnPixWidth</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="4" >
      <widget class="QSpinBox" name="spnPixHeight" >
       <property name="suffix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>8192</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="2" column="2" >
      <widget class="QSpinBox" name="spnPixWidth" >
       <property name="suffix" >
        <string/>
       </property>
       <property name="maximum" >
        <number>8192</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="label_3" >
       <property name="text" >
        <string>In &amp;Percent</string>
       </property>
       <property name="buddy" >
        <cstring>spnPctWidth</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="4" >
      <widget class="QLabel" name="label_2" >
       <property name="text" >
        <string>Height</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4" >
      <widget class="QSpinBox" name="spnPctHeight" >
       <property name="suffix" >
        <string> %</string>
       </property>
       <property name="maximum" >
        <number>500</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="1" column="2" >
      <widget class="QSpinBox" name="spnPctWidth" >
       <property name="suffix" >
        <string> %</string>
       </property>
       <property name="maximum" >
        <number>500</number>
       </property>
       <property name="minimum" >
        <number>1</number>
       </property>
       <property name="value" >
        <number>100</number>
       </property>
      </widget>
     </item>
     <item row="0" column="2" >
      <widget class="QLabel" name="label" >
       <property name="text" >
        <string>Width</string>
       </property>
      </widget>
     </item>
     <item rowspan="2" row="1" column="1" >
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>8</width>
         <height>51</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>21</width>
       <height>70</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="btnOk" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>OK</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCancel" >
     <property name="focusPolicy" >
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="text" >
      <string>Cancel</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <resources>
  <include location="resize.qrc" />
 </resources>
 <connections/>
</ui>
